### PR TITLE
Realtime updates

### PR DESCRIPTION
**Pull request by fredrikved**
Post realtime updates to a firestore.

Infra not ready.

### CODE REVIEW

File: backend/cmd/jobs/env.go [hunk 1]
```diff
@@ -35,6 +35,10 @@ type crowdinConfig struct {
 	ProjectIDs []int
 }
 
+type firebase struct {
+	ProjectID string
+}
```
Comment (KillerX) [inline backend/cmd/jobs/env.go:40]: Can it implicitly use "current project id" if not set when running in GCP?

**Review by KillerX — COMMENTED** [review]
Looks good imo, <del>maybe we should disable it if the PROJECT ID is not there?</del> Never mind, I see it.

File: backend/cmd/jobs/env.go (hunk 1 above)
Comment (fredrikved) [reply backend/cmd/jobs/env.go:40]: Probably, which variable is that stored in?

**Review by KillerX — APPROVED** [review]

**Comment by fredrikved** [comment]
Just merging. Shouldn't affect anything yet anyway.

**Pull request merged**